### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_CreateNewProject_Keyboard : The tooltip for controls such as "Create New Project", "Create Function", etc. are not visible using keyboard.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2287]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#E2E-AzureExtensionsForVSCode-Mac-Aug2020;#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yMAS;#MAC;#FTP;#Benchmark;#Keyboard;#A11ySev2;#WCAG2.1.1;#WCAG1.1.1;#External: VSCode

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code
2. Click on the extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button.
4. Navigate to the toolbar with the controls such as "Create New Project", "Create Function", etc. using a keyboard.
5. Observe if the tooltip content of these controls is visible when the keyboard focus to these controls.

### Actual:
When keyboard users navigate to the controls such as "Create New Project", "Create Function", etc. using the 'Tab' key the tooltip content is not available for keyboard users which is visible when the mouse hovered over these controls which is incorrect.

### Expected:
The tooltip for controls such as "Create New Project", "Create Function", etc. should be available for the keyboard users also when they navigate to these controls.

### User Impact:
If the tooltip content available for the controls on the screen is not available fo the keyboard only users then they will miss out on the information that is available for the mouse users.

### Attachment link for Reference:
[2287_A11y_AzureExtensionsforVSCode_AzureFuction_CreateNewProject_Keyboard The tooltip for controls such as "Create New Project", "Create Function", etc. are not visible using keyboard.zip](https://github.com/microsoft/vscode-azurefunctions/files/5042509/2287_A11y_AzureExtensionsforVSCode_AzureFuction_CreateNewProject_Keyboard.The.tooltip.for.controls.such.as.Create.New.Project.Create.Function.etc.are.not.visible.using.keyboard.zip)

**Comment by EricJizbaMSFT** [comment]
Duplicate of https://github.com/microsoft/vscode-azurefunctions/issues/2228

**Comment by EricJizbaMSFT** [comment]
VS Code has said they will not fix it:

> As per my comment [above](https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443) and because none of our users ever requested this I am closing this as designed. We will not change this.

Closing for that reason and due to lack of response from accessibility test team on how to get this off our tracking list.
